### ISSUE TITLE
Cinnamon 17.3 update is breaking my system

### ISSUE THREAD

**Issue by corinnewelsh** [linuxmint/Cinnamon#5218]
**Former installation:** Linux Mint Maya LTS Cinnamon 64-bit.
**Result:** ran successfully.

**New required install:** Linux Mint 17.3 (preferable LMDE 2 Betsy).
**Result:** no usable cinnamon desktop. No panel, no icons, no start menu, no mouse or keyboard support.

**Reproducible:** Live version of Linux Mint 17.3 Rosa fails -- tried both 32- and 64-bit iso (downloaded 16/04/16). Live version of LMDE 2 Betsy Cinnamon 64-bit (downloaded 16/04/16) runs and installs, but the first update breaks it.

**Alternative successful install:** Any current Linux Mint version with Mate DE.

**Tentative conclusion:** the accelerated Cinnamon DE is broken by the latest update. The non-accelerated Mate DE is unaffected. The issue appears to lie with a conflict between the latest Cinnamon DE updates and the Radeon HD 6310 driver.

**Summary:** Cinnamon 64-bit DE used to work with the Radeon HD 6310 but now it does not. Mate 64-bit DE is unaffected.

**Comment:** surely any new updates to the Cinnamon 64-bit DE must be backwardly compatible and not break the DE.

**System:** (output from inxi -F)
System:    Host: monet Kernel: 3.16.0-4-amd64 x86_64 (64 bit) 
           Desktop: MATE 1.12.0  Distro: LinuxMint 2 betsy 
Machine:   Mobo: ZOTAC model: HUDSON-M1
           Bios: American Megatrends v: 4.6.4 date: 06/16/2011
CPU:       Dual core AMD E-350 (-MCP-) cache: 1024 KB 
           Clock Speeds: 1: 1280 MHz 2: 800 MHz
Graphics:  Card: Advanced Micro Devices [AMD/ATI] Wrestler [Radeon HD 6310]
           Display Server: X.Org 1.16.4 drivers: ati,radeon (unloaded: fbdev,vesa)
           Resolution: 1680x1050@59.95hz, 1680x1050@59.95hz
           GLX Renderer: Gallium 0.4 on AMD PALM GLX Version: 3.0 Mesa 10.3.2
Audio:     Card-1 Advanced Micro Devices [AMD/ATI] Wrestler HDMI Audio 
           driver: snd_hda_intel 
           Card-2 Advanced Micro Devices [AMD/ATI] SBx00 Azalia (Intel HDA) 
           driver: snd_hda_intel 
           Sound: Advanced Linux Sound Architecture v: k3.16.0-4-amd64
Network:   Card-1: Realtek RTL8111/8168/8411 PCI Express Gigabit Ethernet Controller
           driver: r8169
           IF: eth0 state: up speed: 100 Mbps duplex: full 
           mac: 00:01:2e:35:d9:66
           Card-2: Qualcomm Atheros AR9285 Wireless Network Adapter (PCI-Express)
           driver: ath9k
           IF: wlan0 state: down mac: e0:b9:a5:60:3f:10
Drives:    HDD Total Size: 151.5GB (26.8% used)
           ID-1: /dev/sda model: SanDisk_SDSSDX12 size: 120.0GB
           ID-2: USB /dev/sdb model: DataTraveler_2.0 size: 31.4GB
Partition: ID-1: / size: 103G used: 5.4G (6%) fs: ext4 dev: /dev/sda6 
           ID-2: /boot size: 92M used: 44M (51%) fs: ext2 dev: /dev/sda1 
           ID-3: swap-1 size: 8.39GB used: 0.00GB (0%) fs: swap dev: /dev/sda5 
Sensors:   System Temperatures: cpu: 77.4C mobo: N/A gpu: 77.0 
           Fan Speeds (in rpm): cpu: N/A 
Info:      Processes: 159 Uptime: 4:24 Memory: 1000.0/7607.2MB 
           Client: Shell (bash) inxi: 2.1.28

**Comment by Ufeldraku** [comment]
Try this, 
 `sudo nano  /etc/X11/xorg.conf.d/20-radeon.conf`
Then paste this 
`Section "Device"
        Identifier  "Radeon"
        Driver "radeon"
        Option "AccelMethod" "Glamor"
        Option "DRI" "3"
        Option "ColorTiling" "on"
        Option "ColorTiling2D" "on"
EndSection

`
Then reboot, and try normal Cinnamon.

**Comment by Ufeldraku** [comment]
Give 5 or 6 blank spaces to the lines between Section and Endsection

**Comment by two-dogs** [comment]
@corinnewelsh did you try the fglrx driver? regardless of DE the install via driver-manager should be doable, i would also get off of the 3.16, while on it my machine acted most broke, i suggest at least 4.2.xx, our bios dates are aprox, so i dont see an issue there.

**Comment by corinnewelsh** [comment]
Thank you both for your suggestions.  I managed to fix the issue by installing Linux Mint Debian and installing the fglrx driver prior to doing any upgrades.  (Thanks again to two-dogs for that suggestion.) 

There seemed to be no way of installing the fglrx driver with Linux Mint 17.3 Rosa (or using the live version) because it defaulted to a blank desktop and no access to the terminal.  By contrast, whilst troubleshooting, I tried the latest pure Debian operating system, which defaults to a restricted desktop allowing basic access.  It would be better if Linux Mint used a similar process of a restricted desktop to allow for driver installation.
